.pre-commit-config.yaml

```
@@ -0,0 +1,29 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+files: ^app/
+exclude: (^app/core/migrations/|^app/tests/data/|^app/core/management/commands/data/)
```

```suggestion
exclude: (^app/core/migrations/|/tests/data/|^app/core/management/commands/data/)
```
would that exclude all tests/data directories?

---

I think so, I will test it!